apps/s_server.c

```diff
@@ -2262,11 +2262,10 @@ static int sv_body(int s, int stype, int prot, unsigned char *context)
             BIO_ctrl(sbio, BIO_CTRL_DGRAM_MTU_DISCOVER, 0, NULL);
```

Comment:
While you are at it, it might be appropriate to fix "if-if-has-{}-then-else-has-to-have-pair-too" styling rule.